index.html

```diff
@@ -30,7 +29,7 @@
           />
         </a>
 
-        <div>
+        <nav class="nav">
           <ul class="navigation__list">
             <li class="navigation__item">
               <a class="navigation__link" href="#">Home</a>
```

Comment:
When user hover on these links. the background should be white and the text color should be orange
![Screenshot 2023-09-27 at 10 34 07](https://github.com/CodeYourFuture/bikes-for-refugees/assets/13808341/a07c2007-2456-4ecc-9572-247783c72607)